format/spec.md

```diff
@@ -1370,3 +1370,16 @@ Writing v2 metadata:
     * `sort_columns` was removed
 
 Note that these requirements apply when writing data to a v2 table. Tables that are upgraded from v1 may contain metadata that does not follow these requirements. Implementations should remain backward-compatible with v1 metadata requirements.
+
+## Appendix F: Implementation Notes
```

Comment:
I am a bit hesitant on adding an "implementation notes" section to the spec. 

I am wondering if time travel clarification should be added to the `snapshots` section of the spec?

Reply:
I can move it there.

Reply:
I like this approach. I don't think that the spec mandates this time travel behavior, but it is what implementations should do to be consistent with one another. Seems like "Implementation notes" is a good place for this kind of thing.

Reply:
@stevenzwu @rdblue @ajantha-bhat  not sure if either of you have more strongly held preference on this, happy to do it either way.

Reply:
Maybe we should call it Implementation Suggestions, or Suggested Behaviors?

I'm fine with it being a separate section

Reply:
I can see Ryan's point. I can withdraw my argument.

Reply:
Just wondering why can't we mandate this in spec? because it is optional field? We can use "if exists" while mandating in the spec?

Also, we have implementation suggetions like  "File System Tables" and "Metastore tables" in the spec. So, can we have a "Time Travel" section for this?

Reply:
@ajantha-bhat I think the two sections you described above are required for writers/readers for proper functionality.  As per above this is not required for correctness but allows for consistency on implementation.